File: fullstack-fintech-app/bank-collection-service/dto/user_dto.go
package dto

type UserDTO struct {
	ID        int64           `json:"id"`
	Email     string          `json:"email"`
	Username  string          `json:"username"`
	Enabled   bool            `json:"enabled"`
	Records   []UserRecordDTO `json:"records,omitempty"` 
	Role      string          `json:"role,omitempty"`    
}

File: resiliences-service/DTOs/UserDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace resiliences_service.DTOs
{
    public class UserDTO
    {
        public long Id { get; set; }
        public string Email { get; set; } = default!;
        public string Username { get; set; } = default!;
        public bool Enabled { get; set; }
        public List<UserRecordDTO> Records { get; set; } = new List<UserRecordDTO>();
    }
}